apps/charterafrica/src/lib/openAfrica.js

```diff
@@ -117,9 +119,35 @@ export default async function fetchDatasets(organization, query = {}) {
 
   try {
     const response = await packageSearch(params);
-    const formattedData = formatResponse(response);
+    const formattedData = formatResponse(response, path);
     return formattedData;
   } catch (error) {
     return error;
   }
 }
+
+export async function fetchDataset(id, path) {
```

Comment:
```suggestion
export async function fetchDataset(id, pathname) {
```

[`path`](https://nodejs.org/api/url.html#urlobjectpath) is some old school pathname + query or something like that.